zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -94,8 +94,8 @@
         <li ng-if="!ticket">
             <button class="btn btn-default" data-toggle="modal" data-target="#loginModal" ng-click="showLoginWindow()" style="margin-left: 10px">Login</button>
         </li>
-        <li ng-show="ticket.principal != 'anonymous'">
-        	<button type="button" class="btn btn-default btn-xs" ng-click="logout()" tooltip-placement="bottom" tooltip="logout">{{ticket.principal}}</button>
+        <li ng-show="ticket.principal && ticket.principal!='anonymous'" style="left: 5px;">
+        	<button class="btn btn-default" ng-click="logout()" tooltip-placement="bottom" tooltip="logout">Logout</button>
```

Comment:
Agreed if logout button was coming in a separate menu (or different dropdown menu). Right now the UI already says `<li>{{truncatedUsername}}</li>
<li><button>Logout</button></li>`